source/chapters/configuration.rst

```diff
@@ -204,10 +206,95 @@ designed for demanding low latency audio programs like Mixxx. It can be
 difficult to setup JACK and PulseAudio to work well together. So, unless you
 already use JACK, it is easiest to let Mixxx suspend PulseAudio and use ALSA.
 
-.. warning:: On GNU/Linux it is not recommended to use the ``pulse`` device
-             with the ALSA Audio API. This is an emulation layer for ALSA
-             provided by PulseAudio. It has poor performance, but may be
-             suitable for basic playback.
+If the PulseAudio plugin for alsalibs is installed on GNU/Linux, you can 
+choose the virtual device ``pulse``. This allows Mixxx to share the default 
+system sound card with other media players. This only works if you start 
+mixxx without pasuspender, which you can do by running "mixxx" from a console 
+rather than clicking the launcher icon in a menu or on your desktop. Since the 
+sound stream is routed from ALSA to Pulse and back to ALSA, this adds an 
+additional latency of ~2 x the selected audio buffer.  
+ 
+Equalizer Preferences
+=====================
+
+.. sectionauthor::
+   Daniel Schürmann <daschuer@mixxx.org>
+
+.. figure:: ../_static/Mixxx-111-Preferences-Equalizer(TODO).png
+   :align: center
+   :width: 80%
+   :figwidth: 100%
+   :alt: Equalizer Preferences
+   :figclass: pretty-figures
+
+   Equalizer Preferences
+
+:menuselection:`Preferences --> Equalizer` allows you to setup the equalizers.
+
+* **Equalizer Rack**: The Equalizer Rack is a special Effect Rack that is
+  connected to the deck's equalizer and filter controls.
+
+  In this section you can select the equalizers and quick effects that are used
+  with the decks.
+
+* **Equalizer Plugin**: Here you can select the effect that is used as the mixing
+  EQ in each deck. By default only built-in equalizers are displayed. If you uncheck
+  "Only allow EQ knobs to control EQ specific effects" you can select any other
+  effect.
+
+* **Quick Effect**: Here you can select the effect that is controlled by the
+  dedicated filter knob in each deck. By default only built-in filter effects are
+  selected for all decks, but that can be changed as above.
+
+* **High/Low Shelf EQ**: This slider sets the crossover frequencies of the mixing 
+  EQ. It controls which frequency range is affected by the Low, Mid, and High 
+  channel EQ knobs. By default the low knob controls the bass and sub bass 
+  range up to 246 Hz. The mid knob controls the mid range up to 2,5 kHz. 
+  The remaining teble range is controlled by the high knob.   
+
+* **Master EQ**: This section allows you to setup an EQ that effects only the
+  master output. 
+
+
+Mixing Equalizers
+-----------------
+
+Mixxx offers three types of mixing equalizers with full kill feature. They are 
+actually isolators, adopted from analog cross over networks. Each EQ is 
+combination of a high shelf filter, a band pass filter, and a low shelf filter. 
+All EQ types are offering its own individual sound, so try them out. 
```

Comment:
This sentence is a bit awkward. How about:
Each EQ type has its own sound, so try them out to find which one you prefer.